src/Controls/tests/UITests/Tests/Issues/Issue19831.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium.MultiTouch;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue19831 : _IssuesUITest
+{
+	public override string Issue => "[Android] Action mode menu doesn't disappear when switch on another tab";
+
+	public Issue19831(TestDevice device)
+		: base(device)
+	{ }
+
+    [Test]
+	public void ActionModeMenuShouldNotBeVisibleAfterSwitchingTab()
+	{
+		this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
+		var rect = App.WaitForElement("Item1").GetRect();
+		var app = App as AppiumApp;
+
+		if (app is null)
+			return;
+
+		var actions = new TouchAction(app.Driver);
```

Comment:
This test requires changes, we are using now the latest appium version where TouchAction is deprecated. Must use `ActionSequence`. Can find several samples in the repo. Let me know if need help or want me to update it.
```
D:\a\_work\1\s\src\Controls\tests\UITests\Tests\Issues\Issue19831.cs(28,21): error CS0618: 'TouchAction' is obsolete: 'TouchAction is deprecated, please use W3C actions instead: http://appium.io/docs/en/commands/interactions/actions/' [D:\a\_work\1\s\src\Controls\tests\UITests\Controls.AppiumTests.csproj::TargetFramework=net8.0]
    1 Error(s)
```

Reply:
Done!